app/helpers/dates_helper.rb

```diff
@@ -1,16 +1,20 @@
 module DatesHelper
-  class FormatDateError < RuntimeError; end
+  def format_time_to_datetime_at(time)
+    return "" if time.nil?
```

Comment:
does it have to return an empty string?

I would return nil and write this line:

`return unless time`

But that is just a personal preference